lang/cy.json

```diff
@@ -554,7 +554,7 @@
         "many": "Enwau’r atwrneiod wrth gefn",
         "other": "Enwau’r atwrneiod wrth gefn"
     },
-    "readDonorNameLpa": "Darllen LPA {{ .DonorFullName }}",
+    "readDonorNameLpa": "Darllen LPA {{possessive .DonorFullName}}",
```

Comment:
`possessive` doesn't exist for the Welsh translations, do you need it to work here with English style rules?

Reply:
Nope, sorry I thought it needed to have the same bracketed text. It's fine without.